ddl/placement/bundle.go

```diff
@@ -86,44 +93,59 @@ func NewBundleFromConstraintsOptions(options *model.PlacementSettings) (*Bundle,
 			return nil, fmt.Errorf("%w: LeaderConstraints conflicts with Constraints", err)
 		}
 	}
-	rules = append(rules, NewRule(Leader, 1, leaderConstraints))
+	leaderReplicas, followerReplicas := uint64(1), uint64(2)
+	if followerCount > 0 {
+		followerReplicas = followerCount
+	}
+	if !needCreateDefault {
+		if len(leaderConstraints) == 0 {
+			leaderReplicas = 0
+		}
+		if len(followerConstraints) == 0 {
+			if followerCount > 0 {
+				return nil, fmt.Errorf("%w: specify follower count without specify follower constraints", ErrInvalidPlacementOptions)
```

Comment:
how about " specify follower count without specify follower constraints when specify other constraints"?
because we can specify follower count only.